python/lsst/ts/GenericCamera/driver/zwocamera.py

```diff
@@ -1349,7 +1355,7 @@ def getImageSize(self):
             bytesPerPixel = 1
         binWidth = int(width / bin)
         binHeight = int(height / bin)
-        return width * height * bytesPerPixel
+        return binWidth * binHeight * bytesPerPixel
```

Comment:
About this... Now that I look at it I remember this.... This method is supposed to return the full image size irrespective of the binning. If you look at the previous history you will see that the original developer thought the same thing as you did, but it is a wrong assumption. The correct one is indeed `return width * height * bytesPerPixel`.

Maybe the docstring should be updated to say something like:

```
Gets sensor full size, regardless of current configuration.
```

Reply:
OK done. But then I removed the computation of binWidth and binHeight because flake8 complains about declaring but not using them.